packages/metrics/src/features/metrics/metricsGenerator.ts

```diff
@@ -243,7 +251,14 @@ export async function fetchRegWithinTimeFrames(
     )
   }
 
-  const timeFramePoints = await query(queryString)
+  const timeFramePoints = await query(queryString, {
+    placeholders: {
+      timeStart,
+      timeEnd,
+      locationId,
+      registrationTargetInDays
```

Comment:
I couldn't seem to find `registrationTargetInDays` in the query?

Reply:
Well noticed!